src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Publish.targets

```
@@ -86,9 +86,14 @@
   </Target>
 
   <Target Name="CopyNativeBinary" AfterTargets="Publish">
-    <!-- override apphost with binary we generated during native compilation -->
+    <!-- replace apphost with binary we generated during native compilation -->
     <Delete Files="$(PublishDir)\$(TargetName)$(NativeBinaryExt)" />
     <Copy SourceFiles="$(NativeOutputPath)$(TargetName)$(NativeBinaryExt)" DestinationFolder="$(PublishDir)" />
+
+    <!-- replace native symbol file if it exists -->
+    <Delete Files="$(PublishDir)\$(TargetName)$(NativeBinaryExt)$(NativeSymbolExt)2" />
```

Is the `2` a typo?

---

Ah, it's a leftover. I was testing whether `DeleteTask` errors for non-existent file so I could skip the condition. Turned out DeleteTask is idempotent while `CopyTask` is not (which is why it has the `Exists` condition).
```suggestion
    <Delete Files="$(PublishDir)\$(TargetName)$(NativeBinaryExt)$(NativeSymbolExt)" />
```